runtime/src/lib.rs

```diff
@@ -294,14 +294,90 @@ parameter_types! {
 	pub const ChainId: u64 = 43;
 }
 
+pub struct MoonbeamPrecompiles;
+
+/// Linear gas cost
+fn ensure_linear_cost(
+	target_gas: Option<usize>,
+	len: usize,
+	base: usize,
+	word: usize
+) -> Result<usize, frame_evm::ExitError> {
+	let cost = base.checked_add(
```

Comment:
Formatting looks wrong

Reply:
Apparently github is failing at showing this correctly.